### PR TITLE
[1LP][RFR][NOTEST] Removed already automated SSA manual tests.

### PR DESCRIPTION
**Pull request by sbulage**
## Purpose or Intent

Here is MR Link--> https://gitlab.cee.redhat.com/cfme-qe/cfme-qe-yamls/merge_requests/877
- [x] SSA manual tests cleanup for SCVMM provider.

### CODE REVIEW

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 1]
```diff
@@ -718,103 +432,42 @@ def test_ssa_vm_azure_windows2016_refs():
...
-        initialEstimate: 1/3h
-        startsin: 5.9
-        tags: smartstate
-    """
-    pass
-
-
-def test_ssa_vm_azure_wimdows2012_ssui():
+def test_ssa_vm_azure_wimdows2012_ssui():
     """
```
Comment (ganeshhubale) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: is this typo: *```wimdows2012```*

**Comment by sbulage** [comment]
> LGTM :)
> 
> You have updated some tests. So please add it's `testSteps` and `expectedResults`

I haven't updated tests, only removed already automated tests.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py (hunk 1 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: I will fix this typo. Good Catch :wink:

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 2]
```diff
@@ -718,107 +432,46 @@ def test_ssa_vm_azure_windows2016_refs():
...
+        caseimportance: medium
+        initialEstimate: 1/3h
+        startsin: 5.9
         tags: smartstate
     """
     pass
 
 
-def test_ssa_vm_azure_region():
+def test_ssa_vm_azure_windows2012_ssui():
```
Comment (mshriver) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: I would like to see a pass at parametrizing the remaining test functions here, as there are multiple azure tests on different windows versions. That should be parameters, not test function names.

For a followup PR, not this one :)

Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: Sure I will update it.
Thanks for the suggestion.

**Pull request merged**